augly/tests/video_tests/transforms/composite_test.py

```diff
@@ -23,6 +26,16 @@ def setUpClass(cls):
     def test_ApplyLambda(self):
         self.evaluate_class(vidaugs.ApplyLambda(), fname="apply_lambda")
 
+    def test_AugmentAudio(self):
+        self.evaluate_class(
+            vidaugs.AugmentAudio(
+                audio_aug_function=audaugs.Compose(
+                    [audaugs.PitchShift(), audaugs.Reverb()]
+                ).__call__(sample_rate=DEFAULT_SAMPLE_RATE),
```

Comment:
You want to pass in a **function** here; if you add this `(sample_rate=DEFAULT_SAMPLE_RATE)` then you're calling the function before even passing it in.
```suggestion
                ).__call__,
```